run.sh

```diff
@@ -270,6 +272,7 @@ clean_up() {
     fi
     if is_dir "${SMALLTALK_CI_BUILD_BASE}"; then
       print_info "  ${SMALLTALK_CI_BUILD_BASE}"
+      chmod -fR +w "${SMALLTALK_CI_BUILD_BASE}"
```

Comment:
The GemStone download is chock full of read only files and the `chmod` is required before `rm`ing the `_build` directory. There are no read only GemStone artifacts in the cache directory so it isn't required ...